nbexchange/plugin/fetch_feedback.py

```diff
@@ -25,11 +27,19 @@ def init_dest(self):
 
     def download(self):
         self.log.debug(f"Download feedback for {quote_plus(self.coursedir.notebook_id)} from {self.service_url}")
-        r = self.api_request(
-            f"feedback?course_id={quote_plus(self.coursedir.course_id)}&assignment_id={quote_plus(self.coursedir.assignment_id)}"  # noqa: E501
-        )
+        try:
+            r = self.api_request(
+                f"feedback?course_id={quote_plus(self.coursedir.course_id)}&assignment_id={quote_plus(self.coursedir.assignment_id)}"  # noqa: E501
+            )
+        except requests.exceptions.Timeout:
+            self.fail("Timed out trying to reach the exchange service to fetch feedback.")
```

Comment:
This error message is not unique. We'll probably get a stack trace if it happens, but it'd be good if they were all unique.

Reply:
just double checked I wasn't catching a test usage, I'm not - it also comes up in noteable_auth.py